variables.tf

```diff
@@ -167,6 +167,21 @@ variable "reserved_concurrent_executions" {
   }
 }
 
+variable "triggering_sns_arns" {
+  description = "List of SNS ARNs. If provided a SQS for triggering the Lambda will be added and subscribed to the SNS."
```

Comment:
"List of SNS ARNs" is not explanatory enough. describe the content of the sns or the usecase for which the sns arns should be provided.